src/inputs/Table/Table.js

```diff
@@ -374,6 +403,21 @@ Table.propTypes = {
    * If not defined, this button will not be rendered.
    */
   onExport: PropTypes.func,
+  /*
+   * Callback function that will be called when user click on the add row button.
+   * If not defined, this button will not be rendered.
+   */
+  onAddRow: PropTypes.func,
+  /*
+   * Callback function that will be called when user click on the delete row button.
+   * If not defined, this button will not be rendered.
+   */
+  onDeleteRow: PropTypes.func,
+  /*
+   * Boolean which define if you wan't to had the addRow features.
+   * Need to be disabled if you wan't to had some non editable columns.
```

Comment:
typo: which **defines whether you want or not to enable** addRow featur**e**. Need to be disabled if you **want to have** some..

Reply:
Do not exist anymore, DONE